src/promptflow/promptflow/_sdk/schemas/_run.py

```diff
@@ -34,6 +36,23 @@ class ResourcesSchema(metaclass=PatchedSchemaMeta):
     compute = fields.Str()
 
 
+class ManagedIdentitySchema(metaclass=PatchedSchemaMeta):
+    type = StringTransformedEnum(
+        required=True,
+        allowed_values=[IdentityConfigurationType.MANAGED, IdentityConfigurationType.MANAGED_IDENTITY],
```

Comment:
LOL, looks we also bring the history of v2 here :)

Reply:
Only supported "managed" as offline synced.